### ISSUE TITLE
Empty queryType and mutationType

### ISSUE THREAD

**Issue by terion-name** [ardatan/graphql-import#191]
I split my schema into pieces. Then I import them piece by piece, make some processing/generatoin and then merge them all together.

Having such piece:
```graphql
# import Node from "../../schema/generated/mysql.graphql"
# import DocumentTranslated from "../document/schema.graphql"

type AttachmentTranslated implements Node {
    id: ID!
    createdAt: DateTime!
    updatedAt: DateTime!
    file: File
    title: String!
    document: DocumentTranslated
}
```

It does not define Query and Mutation, because this type should not be queried from root

After importing the schema, it has empty queryType and mutationType, that leads to errors on further merging.

```
GraphQLSchema {
  __allowedLegacyNames: undefined,
  _queryType: null,
  _mutationType: null,
  _subscriptionType: null,
  _directives:

```

I think that due to intention of this package (splitting schema in reusable pieces), this types should be populated automatically, shouldn't they?

**Comment by the-vampiire** [comment]
we had this working on gql-import v0.4.0 and it is now broken upgrading to v0.6.0

any update on this or a potential fix? I am going through the codebase migrating to apollo server 2.0 and upgraded the ancillary packages leading to this error. It says my queries and mutations are not defined in the schema.

**Comment by SpaceK33z** [comment]
Would it be possible for any of you to make a full example reproducing this scenario?

**Comment by ardatan** [comment]
Hi @ma-jahn !
In 1.0.0 beta release, we introduced a lot of changes including schema definition support (schema definition with `query`, `mutation` and `subscription` assignments;
Could you install `graphql-import@beta` to try new changes? Don't forget to modify your code regarding to the migration notes in `README`. 
https://github.com/ardatan/graphql-import#updating-from-07x

You can also get the final schema in different formats such as `string`, `GraphQLSchema` and `DocumentNode`;

```ts
const finalSchema = await importSchema('./some-schema.graphql', {}, 'DocumentNode');
```

**Comment by ardatan** [comment]
Available in 1.0.0!
